packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -215,3 +215,18 @@ export function computeSize(entry: RumPerformanceResourceTiming) {
 export function isAllowedRequestUrl(configuration: RumConfiguration, url: string) {
   return url && !configuration.isIntakeUrl(url)
 }
+
+const DATA_URL_REGEX = /data:(.+)?(;base64)?,/g
+export const MAX_ATTRIBUTE_VALUE_CHAR_LENGTH = 24_000
+
+export function isLongDataUrl(url: string): boolean {
+  if (url.length <= MAX_ATTRIBUTE_VALUE_CHAR_LENGTH) {
+    return false
+  }
+  const match = url.match(DATA_URL_REGEX)
+  return !!match
+}
+
+export function sanitizeDataUrl(url: string): string {
+  return url.match(DATA_URL_REGEX)![0]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**
It could be nice to have something indicating that it the url has been truncated like `data:[<mediatype>] [...]` (the same way we do for[ action names](https://github.com/DataDog/browser-sdk/blob/160d5fef65a055cebae3fbc50539b26d3d1075c9/packages/rum-core/src/domain/action/getActionNameFromElement.ts#L161))

Reply:
I've added the [...] as in action names to indicate the truncation.